x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.scss

```
@@ -0,0 +1,9 @@
+.relevanceTuningForm {
+  .relevanceTuningForm__item {
```

This likely doesn't need to be an override

```suggestion
  &__item {
```

---

So, I should have called that out. I'll defer to whatever you prefer, but using `&__item` feature in sass trips me up more than it helps.

It's like, whenever I want to figure out where a class is used, I do a test search for the class name ... i mean afterall, it's not like we have decent linking like we get in something like TS. 

When classnames are divided up like this it just makes me a bit disoriented and I tend to miss declarations. Like I search for "relevanceTuningForm__item" and don't find the declaration, etc.

I won't object to making this change though, if you prefer it.

---

I'll admit I'm very Sass-centric so `&` use feels like a central feature of the language to me, but I definitely see the issue with searching for full names. However, we should still avoid unnecessary nested selectors where possible by instead doing something like this:

```scss
.someParent {
 // etc
}
  .someParent__someChild {
    // etc
  }
```

So that way we generate flat selectors - `.someParent__someChild` and not `.someParent .someParent__someChild`, which becomes more difficult to override if we need to.

---

Ah, I see your point. Yeah, I will update it.

---

https://github.com/elastic/kibana/pull/90621/commits/805f5afc952c6ab0dc297308aa9b813c8629aeaf

---

I totally didn't mean to bully you into using `&`, sorry haha 😬 Definitely feel free to explicitly spell out the full CSS class in the future but keep it un-nested!